src/dk.nita.saml20/dk.nita.saml20/Saml20MetadataDocument.cs

```diff
@@ -195,6 +195,11 @@ private void ConvertToMetadata(SAML20FederationConfig config, IEnumerable<KeyInf
 
                 keyEncryption.use = KeyTypes.encryption;
                 keyEncryption.useSpecified = true;
+                keyEncryption.EncryptionMethod = new Schema.XEnc.EncryptionMethod[]
+                {
+                    new Schema.XEnc.EncryptionMethod{Algorithm="http://www.w3.org/2001/04/xmlenc#aes256-cbc" },

```

Comment:
Use Saml20Constants.XENC as prefix for both encryption methods and maybe add the encryption mehtods as constants as well !